src/actions/web3.js

```diff
@@ -21,12 +22,15 @@ export const web3Connection = () => async (dispatch, getState) => {
 
 export const readSmartContract = address => async (dispatch, getState) => {
   const web3 = getState().modal.web3;
-
   console.log(`Reading Smart contract , RPC connection ${web3 ? 'Connected' : 'Disconnected'}`);
   if (!web3) {
     return;
   }
-  const answers = config.ICOs[address].matrix;
+  let configFile = config.ICOs;
+  if (process.env.NODE_ENV === 'test') { configFile = testConfig.ICOs; }
```

Comment:
this is not perfect but i think we need to live with it for now. The other possibility would be to use https://github.com/speedskater/babel-plugin-rewire